index.js

```diff
@@ -191,12 +213,16 @@ FocusGroup.prototype._findIndexOfNode = function(searchNode) {
 };
 
 FocusGroup.prototype._getActiveElementIndex = function() {
-  return this._findIndexOfNode(document.activeElement);
+  return this._rootNode !== document
+    ? this._activeIndex
+    : this._findIndexOfNode(document.activeElement);
 };
 
 FocusGroup.prototype.focusNodeAtIndex = function(index) {
   var member = this._members[index];
   if (member) focusNode(member.node);
```

Comment:
Not sure what you mean here? It is making the assumption that anything other than `document` will mean you want a fake focus group. I guess I'm not understanding what the extra option would be? `fauxFocus` or something?

My idea was that you need to have something focused in order to control the fake focus group. So you provide your root node which will allow you to control the focus group once it has received focus. Which could be a parent element or even a button or input.